### PR TITLE
Solved #152 issue

### PR DESCRIPTION
**Pull request by munnokd**

### CODE REVIEW

**Comment by munnokd** [comment]
Hey, @gbowne1  I solved the #152 issue please merge this pull request.

**Comment by pawel975** [comment]
@munnokd I checked changes, and I think it would be better if you would follow general app styling. Try to use any of variables from src/variables.css. Hard-coded value would be hard to manage if general styling would change.

File: src/assets/SideNav.css [hunk 1]
```diff
@@ -30,7 +30,7 @@
   
   /* Color/shape of close button cross */
   .bm-cross {
-	background: var(--bg-light-900);
```
Comment (pawel975) [inline src/assets/SideNav.css:33]: Try to change it on any other value from src/variables.css. There are some accent values for both dark and light modes

Comment (munnokd) [reply src/assets/SideNav.css:33]: okay I will change it,

Comment (gbowne1) [reply src/assets/SideNav.css:33]: This was only applied to the SideNav.  I forgot to mention this is in all of the panels in dark mode and refers to the header of each panel component.

Comment (munnokd) [reply src/assets/SideNav.css:33]: okay

**Comment by pawel975** [comment]
Hi @munnokd ! Do you work on that issue?

**Comment by munnokd** [comment]
> Hi @munnokd ! Do you work on that issue?

Yes I am working on it

**Pull request closed without merging**